Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -152,6 +155,25 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	if (skip_drawing || is_preprocess)
 		return size;
 
+	// Object Removal Code code
+	if (m_LocalCoreStartupParameter.num_object_removal_codes)
+	{
+		if (m_LocalCoreStartupParameter.update)
+		{
+			// Set lock so codes can be enabled/disabled in game without crashes.
+			m_LocalCoreStartupParameter.done = false;
+			for (int current_object_removal_code = 0; current_object_removal_code < m_LocalCoreStartupParameter.num_object_removal_codes; current_object_removal_code++)
```

Comment:
@degasus Good idea.  I'll check this out and check the disassembly/profile it to make sure it's just as fast.  I originally stored the size because I thought it would be faster to calculate the size once and not have to call a function to calculate it over and over again each time through.  I'm not sure if that's a false assumption though.